.eslintrc.js

```diff
@@ -76,8 +76,11 @@ module.exports = {
         'import/no-extraneous-dependencies': [
             'error',
             {
-                // add scripts/ folder
-                devDependencies: ['**/*.test.{tsx,ts,js}'],
+                devDependencies: [
+                    '**/*.test.{tsx,ts,js}',
+                    '**/suite-desktop/scripts/**',
+                    '**/suite-desktop/e2e/**',
```

Comment:
not sure about this @Nodonisko

Reply:
Is there any particular reason fot that? It's probably not that critical for dev scripts etc but we still should aim to have all dependencies specified otherwise it could lead to some strange behavior because it will pick random version of package that is currently available in node_modules etc.

Reply:
The problem there, is that it wants me to move devDependencies into dependencies

Reply:
I see then, this is probably fine.